packages/core/src/lib/Button/ButtonBase/styles/ButtonBaseTest.styles.tsx

```diff
@@ -28,6 +28,7 @@ const textStyles = css`
     background-color: transparent;
     border-width: 0px;
     color: ${color.navy40};
+    padding: 0px;
```

Comment:
i was about to add a customisation prop to Button, just FYI the default is current secondary so you never actually see base button

Reply:
Not sure what you mean by current secondary, I have added that so I can avoid this on Admin to Button of theme text. but adding customization props is more flexible :raised_hands:
```
const ButtonStyled = styled(Button)`
  padding: 0px !important;
`;
```